tests/lean/file_not_found.lean.expected.out

```diff
@@ -0,0 +1,9 @@
+
+file_not_found.lean:6:0: error: No such file or directory
+  file: non-existant-file.txt
```

Comment:
For comparison, this is what Python3 prints:
```python
Traceback (most recent call last):
  File "err.py", line 1, in <module>
    f = open('hello', 'r')
FileNotFoundError: [Errno 2] No such file or directory: 'hello'
```
Note that the line of the error is copied, the error types have names, and the OS codes are provided.

Reply:
> Note that the line of the error is copied, the error types have names, and the OS error code is provided.

We could print the line of the error but we'd need to change `#eval` so that it prints the expression when encountering the error. It's an interesting feature but I think it deserves its own PR.

I can add the OS error code but I don't see the point of printing `FileNotFoundError`. It's just a less readable version of the rest of the error message.